src/scene/index.js

```diff
@@ -94,7 +94,7 @@ function onMouseDown(event) {
   // Handle Event: "Clicked Nothing" (deselect/hide selections)
   if (!intersects || !intersects[0]) {
     clearHighlightedPoint();
-    const selection = window.document.getElementsByClassName('location visible')[0];
+    const selection = window.document.getElementsByClassName('globe-location-card visible')[0];
```

Comment:
Hmm, if you needed to change this to be getting elements with these two classes:
1. `globe-location-card`
2. `visible`

Then I think the Location Marker "Auto-Select" functionality will also need this change. In earth.js, a similar line to this one exists where it is grabbing by `location visible` classes 🐒

Reply:
PS: I am blind, you did this already 🐒 nvm